### ISSUE TITLE
Multiple instances of the same Firefox found

### ISSUE THREAD

**Issue by declension** [litixsoft/karma-detect-browsers#25]
Hi. New to the project, might be misunderstanding something so sorry if so.  
### Steps
* Running on Ubuntu 17.04, fairly standard installation.
* Run the build for a project using `karma-detect-browser`

### Expected
One Firefox instance launched

### Actual
Three (all at `56`, FWIW) are launched. Here's some logging just before
```
INFO [framework.detect-browsers]: The following browsers were detected on your system: [ 'Chrome',
  'Firefox',
  'FirefoxAurora',
  'FirefoxNightly',
  'PhantomJS' ]
```
Thanks

**Comment by 4kochi** [comment]
Fix released in v2.2.6

**Comment by declension** [comment]
Thanks @4kochi !
